src/Polysemy/Embed.hs

```diff
@@ -0,0 +1,29 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Polysemy.Embed
+  ( -- * Effect
+    Embed (..)
+
+    -- * Actions
+  , embed
+
+    -- * Interpretations
+  , runEmbed
+  ) where
+
+import Polysemy
+import Polysemy.Embed.Type (Embed (..))
+
+------------------------------------------------------------------------------
+-- | Given a natural transform from @m1@ to @m2@
+-- run a @Embed m1@ effect by transforming it into a @Embed m2@ effect.
+--
+-- TODO(sandy): @since
+runEmbed
```

Comment:
Let's go with `runEmbedded`

Reply:
Is this the same thing that exists in `Polysemy.IO`? This is admittedly a better place for it.

Reply:
This is the original `runLift` (now renamed to `runEmbed`, because the effect is `Embed`) from #142, which started this pr.

It's not the same thing as `Polysemy.IO.runEmbedded`.

Personally I'm fine with it being `runEmbed` (in accordance with the other effects), but we need to rename the second one if this is going to be `runEmbedded`.